src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  background-color: $white-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  @include hover(transform, scale(1.2));
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 16px 0;
+
+    @include hover(color, #34568b);
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-left: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin: 8px 16px 0 18px;
+  }
+
+  &__review_count {
+    margin: 0;
+    color: $main-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 22px 16px 0;
+  }
+
+  &__price_text {
+    margin: 0;
+    color: $secondary-text-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price_count {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    background-color: $accent-color;
+    border: 1px solid $accent-color;
+    box-sizing: border-box;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $white-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin: 16px;
+
+    @include hover(background-color, $white-color);
+    @include hover(color, $accent-color);
+  }
```

Comment:
You've used the 'hover' mixin effectively to change the background color and text color on hover. However, keep in mind that it's a good practice to also include a 'focus' state for keyboard users. You can usually use the same styles for both 'hover' and 'focus'.